UserListApp/src/components/User/UserList.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import { UserDetails } from "../Interface/UserAction";
+import { userInfo } from "./Data/UserInfo";
+import UserComponent from "./UserData/UserComponent";
+const UserList = () => {
+  const [records, setRecord] = useState(userInfo);
```

Comment:
Why state is defined when its not updated anywhere in the component and data is already defined in other file ?